From 3d368b51e2f035d6b7b29a3b76fd20405e1f0615 Mon Sep 17 00:00:00 2001
From: Mahendiran Balasubramaniam <mahendir@qti.qualcomm.com>
Date: Mon, 8 Dec 2025 12:29:57 +0530
Subject: [PATCH] UPSTREAM: PROTOCOL: iw: Incorrect CF1 configuration for 6G
 80MHz

Currently 80MHz cf1 is incorrectly configured as 6195 for UNII-8.
This is leading to channel switch failure when using iw tool
As a fix, correcting the cf1 configuration in iw utility.

Fixes: None
Patch-work: None
Tag: 6G_REGULATORY
Patch-dependency: None
Link: [PATCH] iw: Incorrect CF1 configuration for 6GHz 80 MHz

Signed-off-by: Mahendiran Balasubramaniam <mahendir@qti.qualcomm.com>
---
 util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util.c b/util.c
index 15f986f..f700dfb 100644
--- a/util.c
+++ b/util.c
@@ -1779,7 +1779,7 @@ int get_cf1(const struct chanmode *chanmode, unsigned long freq)
 	unsigned int bw80[] = { 5180, 5260, 5500, 5580, 5660, 5745,
 				5955, 6035, 6115, 6195, 6275, 6355,
 				6435, 6515, 6595, 6675, 6755, 6835,
-				6195, 6995 };
+				6915, 6995 };
 	unsigned int bw160[] = { 5180, 5500, 5955, 6115, 6275, 6435,
 				  6595, 6755, 6915 };
 	/* based on 11be D2 E.1 Country information and operating classes */
-- 
2.34.1

